### PR TITLE
Deleting from category_group and setting auto_increment

### PR DESCRIPTION
**Pull request by sadlyblue**
When truncating the catalog, it could remove the categories deleted from the table category_group and set the auto_increment the max existing + 1.

### CODE REVIEW

**Comment by MathiasReker** [comment]
What is status on this?

**Comment by MathiasReker** [comment]
Fixed here: https://github.com/PrestaShop/pscleaner/pull/38

File: pscleaner.php [hunk 1]
```diff
@@ -259,6 +259,8 @@ public function truncate($case)
                 $db->execute('DELETE FROM `'._DB_PREFIX_.'category` WHERE id_category NOT IN ('.implode(',', array_map('intval', $id_home)).', '.implode(',', array_map('intval', $id_root)).')');
                 $db->execute('DELETE FROM `'._DB_PREFIX_.'category_lang` WHERE id_category NOT IN ('.implode(',', array_map('intval', $id_home)).', '.implode(',', array_map('intval', $id_root)).')');
                 $db->execute('DELETE FROM `'._DB_PREFIX_.'category_shop` WHERE id_category NOT IN ('.implode(',', array_map('intval', $id_home)).', '.implode(',', array_map('intval', $id_root)).')');
+		$db->execute('DELETE FROM `'._DB_PREFIX_.'category_group` WHERE id_category NOT IN ('.implode(',', array_map('intval', $id_home)).', '.implode(',', array_map('intval', $id_root)).')');
```
Comment (Progi1984) [inline pscleaner.php:262]:
```suggestion
                $db->execute('DELETE FROM `'._DB_PREFIX_.'category_group` WHERE id_category NOT IN ('.implode(',', array_map('intval', $id_home)).', '.implode(',', array_map('intval', $id_root)).')');
```

**Comment by Progi1984** [comment]
Thanks @sadlyblue

**Pull request merged**